cmd/podman/ps.go

```diff
@@ -204,11 +206,14 @@ func psCmd(c *cliconfig.PsValues) error {
 		span, _ := opentracing.StartSpanFromContext(Ctx, "psCmd")
 		defer span.Finish()
 	}
-
 	var (
 		filterFuncs      []libpod.ContainerFilter
 		outputContainers []*libpod.Container
+		watch            bool
 	)
+	if c.Watch > 0 && c.Flags().Changed("watch") {
```

Comment:
Default it to 0 and drop the `Changed()`

Reply:
Also, does `--watch` make any sense with `--latest`? I feel like the container you're watching changes as new containers get created... Or should we look it up once and then only watch that container?

Reply:
done, and making latest and watch exclusive unless someone asks for it